contracts/examples/PullPaymentBid.sol

```diff
@@ -3,14 +3,15 @@ import '../PullPaymentCapable.sol';
 contract PullPaymentBid is PullPaymentCapable {
   address public highestBidder;
   uint public highestBid;
-  
-  function bid() external {
-    if (msg.value <= highestBid) throw;
-    
+
+  function bid() external returns (bool) {
+    if (msg.value <= highestBid) return false;
+
```

Comment:
Returning false here makes `msg.value` to be added to the contract balance, but no bid is placed! This breaks the contract logic. That's why throw is required here